scripts/configure.ps1

```diff
@@ -0,0 +1,28 @@
+
+$ErrorActionPreference = "Stop"
+
+$PYTHON_FILE_SERVER_ROOT='C:\Windows\Temp\python-simple-http-webserver'
+if (Test-Path $PYTHON_FILE_SERVER_ROOT) { 
+	echo "Removing file server root folder $($PYTHON_FILE_SERVER_ROOT)"
+	Remove-Item $PYTHON_FILE_SERVER_ROOT -Force -Recurse 
+}
+C:\CloudifyAgent\Scripts\ctx logger info "Creating HTTP server root directory at $($PYTHON_FILE_SERVER_ROOT)"
```

Comment:
Absolute path will probably not be required as you fixed the ctx not in path issue.